.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,6 +3,17 @@ name: Bug report
 about: Create a report to help us improve
 ---
 
+**What part of Carbon does this bug correlate to?**
+* [ ]  All graphs (axes, legends etc)
```

Comment:
Do we need all graphs option for bug because i do not recall or think about any bug which will be in common for all graphs? Please correct me if i am wrong.

Reply:
I think this goes along with my other question around changing this to something like `Common graph elements (axes, legends...etc.)` instead of using the words `All graphs...`

Reply:
@mraplumb just small correction in here, since pie chart does not contain axes , i think axes won't be the common graph elements.

Reply:
Things like ticks, labels and axis logic are common to all or multiple graphs. That's what I mean by this. But I guess I should rephrase this better.